### PR TITLE
[Datafactory] Fixing ADF generation issues and the build

### PR DESCRIPTION
**Pull request by hvermis**

### CODE REVIEW

**Comment by hvermis** [comment]
@shahabhijeet  This is the PR to fix autorest generation issues. I had to manually modify DynamicSink file to have the build pass, please let me know if I can fix it in some other way.

**Comment by hvermis** [comment]
@shahabhijeet  Also the CI build is still not running

File: .azure-pipelines/mgmt.yml [hunk 1]
```diff
@@ -6,7 +6,6 @@ pr:
     - BuildToolsForSdk
   paths:
     exclude:
-    - sdk/
```
Comment (weshaggard) [inline .azure-pipelines/mgmt.yml]: You can undo the changes in this file. I've fixed the triggering on the DevOps side.

Comment (hvermis) [reply .azure-pipelines/mgmt.yml]: @shahabhijeet

**Comment by hvermis** [comment]
@shahabhijeet  This PR is ready to be reviewed.

**Comment by hvermis** [comment]
@weshaggard Did your changes go in? Can you start a CI build on this? I need to release our SDK.

File: .azure-pipelines/mgmt.yml (hunk 1 above)
Comment (hvermis) [reply .azure-pipelines/mgmt.yml]: @weshaggard , we can't be blocked by this any longer. I am going to bring this back to get CI build running, as we need to release our SDK.

**Review by shahabhijeet — APPROVED** [review]
